language/llama2-70b/README.md

```diff
@@ -63,32 +63,8 @@ Inside the container, set up the environment with `bash build.sh`. This will ins
 CPU-only setup, as well as any GPU versions for applicable libraries like PyTorch.
 
 
-## Get Model
-+ For now, MLCommons is not hosting the checkpoint, so you must first go to [llama2-request-link](https://ai.meta.com/resources/models-and-libraries/llama-downloads/) and make a request, sign in to huggingface (if you don't have account, you'd need to create one). **Please note your authentication credentials** as you may be required to provide them when cloninng below
```

Comment:
The model is available only for MLCommons members through the google form but someone who is not part of MLCommons may be interested in following the old procedure so I'd keep it.

_The MLCommons license for Llama 2 and the attached side letter are for internal distribution within MLCommons Member organizations only. Any organization that gives up its MLCommons membership must destroy any copies of this Llama 2 model, this license, and the side letter._

Reply:
Added back the old instructions.